### ISSUE TITLE
import MergeTicketingClient fails with SyntaxError

### ISSUE THREAD

**Issue by roansong** [merge-api/merge-ticketing-python#6]
version: 1.0.2

**To reproduce:**
```python
import MergeTicketingClient
```
**Traceback:**
```
File "./src/ticketing/controller.py", line 4, in <module>
    import MergeTicketingClient
  File "/root/.local/share/virtualenvs/app-4PlAip0Q/lib/python3.9/site-packages/MergeTicketingClient/__init__.py", line 17, in <module>
    from MergeTicketingClient.api_client import ApiClient
  File "/root/.local/share/virtualenvs/app-4PlAip0Q/lib/python3.9/site-packages/MergeTicketingClient/api_client.py", line 27, in <module>
    from MergeTicketingClient.model_utils import (
  File "/root/.local/share/virtualenvs/app-4PlAip0Q/lib/python3.9/site-packages/MergeTicketingClient/model_utils.py", line 122
    elif self.additional_properties_type is not None:
    ^
SyntaxError: invalid syntax
```

**Cause:**
This line is incorrectly indented: https://github.com/merge-api/merge-ticketing-python/blob/main/MergeTicketingClient/model_utils.py#L116

**Comment by dan-merge** [comment]
Thanks for raising this! Should be fixed if you upgrade to 1.0.3

**Comment by roansong** [comment]
Looks good on v1.0.3. Thanks!
